app/drops/sla_policy_drop.rb

```diff
@@ -0,0 +1,9 @@
+class SlaPolicyDrop < BaseDrop
```

Comment:
Should we add the specs? 
cc @sojan-official

Reply:
done